spec/v2/Makefile

```diff
@@ -1,12 +1,13 @@
 DIAGRAMS := $(patsubst %.mmd,%.svg,$(wildcard diagrams/*.mmd))
 OUTDIR := ../upload/v2
-MMDC := ./node_modules/.bin/mmdc
+MMDC := mmdc
+FILES := $(shell ls)
```

Comment:
Does not look like this variable is referenced anywhere. Would you remove it, please, @raimundo-henriques ?

Reply:
Thank you! I'm sorry, this was something I was trying and forgot to remove.